cassandra-analytics-common/build.gradle

```diff
@@ -24,22 +24,13 @@ plugins {
     id('maven-publish')
 }
 
+apply from: "$rootDir/gradle/common/publishing.gradle"
```

Comment:
the `java` section below should be removed as it's already added in the publishing.gradle file